Graphics/src/Window.cpp

```diff
@@ -261,15 +251,11 @@ namespace Graphics
 			{
 				if(evt.type == SDL_EventType::SDL_KEYDOWN)
 				{
-					if(m_textComposition.composition.empty())
```

Comment:
The commit ddf6647 introduced this check, with the comment "Also Fixed IME input + enter key in song select".
However I had no problem after removing this check.